3.0.0 sdist is missing tests/__init__.py

When attempting to run the test suite from 3.0.0 sdist, I'm getting errors such as the following:

```pytb
______________________________________________ ERROR collecting tests/test_commonmark.py ______________________________________________
ImportError while importing test module '/tmp/mistune-3.0.0/tests/test_commonmark.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_commonmark.py:2: in <module>
    from tests import BaseTestCase, normalize_html
E   ImportError: cannot import name 'BaseTestCase' from 'tests' (unknown location)
```

It seems that `tests/__init__.py` file is missing from the sdist for some reason.

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,0 +1,6 @@
+# include all test files but avoid including .pyc
+recursive-include benchmark *.py *.txt
+recursive-include tests *.json *.html *.md *.py *.txt
+# include documentation sources but not built docs
+graft docs
+prune docs/_build

